node/src/redis-modules/Json.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+import { RedisClient, RedisClusterClient } from "../../";
+import {
+    ConditionalChange,
+    JsonGetOptions,
+    createJsonGet,
+    createJsonSet,
+} from "../Commands";
+export type TRedisClient = RedisClient | RedisClusterClient;
+
+/**
+ * Sets the JSON value at the specified `path` stored at `key`.
+ * @param client - The Redis client to execute the command.
+ * @param key - The key of the JSON document.
+ * @param path - Represents the path within the JSON document where the value will be set.
+ * @param value - The value to set at the specific path, in JSON formatted string.
+ * @param setCondition - Set the value only if the given condition is met (within the key or path).
+ * Equivalent to ['XX' | 'NX'] in the Redis API.
+ * @returns If the value is successfully set, returns OK.
+ * If value isn't set because of `setCondition`, returns null.
+ */
+export async function set(
+    client: TRedisClient,
+    key: string,
+    path: string,
+    value: string,
+    setCondition?: ConditionalChange,
+): Promise<"OK" | null> {
+    const result = client.customCommand(
+        createJsonSet(key, path, value, setCondition),
+    );
+    return result as unknown as "OK" | null;
```

Comment:
this is the nodejs way to cast it?
`return result as "OK" | null;` isn't working?

Reply:
`Conversion of type 'Promise<ReturnType>' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.ts(2352)
`